### PR TITLE
Keybinding testing

### PR DESCRIPTION
**Pull request by jmeijer8**
Added test suites for the Keybinding and KeybindingContainer Classes. <testing

### CODE REVIEW

**Comment by jessetilro** [comment]
Is related to issue #217.
<review

**Comment by dalderliesten** [comment]
Jeroen, as a general notice, please put the Octopull tags on a newline, and not on the git commit character themselves.

<review

File: src/test/java/nl/tudelft/scrumbledore/keybinding/KeybindingContainerTest.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    assertEquals(KeyCode.LEFT, keybindingContainer.getKeybinding(0).getKey(PlayerAction.Jump));
+
+    assertFalse(keybindingContainer.getKeybinding(0).isAssigned(KeyCode.UP));
+
+    assertEquals(KeyCode.UNDEFINED,
+        keybindingContainer.getKeybinding(0).getKey(PlayerAction.MoveLeft));
+  }
+
+  @Test
```
Comment (NielsWarnars) [inline src/test/java/nl/tudelft/scrumbledore/keybinding/KeybindingContainerTest.java]: Please add a JDoc comment here

<testing

**Comment by NielsWarnars** [comment]
Please run CheckStyle, FindBugs and PMD.
I am not going to merge this branch as long as those things are not fixed.
<review

**Comment by NielsWarnars** [comment]
No more CheckStyle/PMD/FindBugs errors present that can be fixed.
<review

**Pull request merged**